### DISCUSSION TITLE
Validation Custom Message for regex rule

### DISCUSSION THREAD

**Discussion by Professor93M** [filamentphp/filament#8099] [Help] [answered]
Hello,

Could you please show me how to customize the validation error message for regex rule?

it always return **validation.regex**

thank you.

**Comment by pboivin** [comment]
Can you share some code to show how you are using the validation rule?

**Reply by Professor93M** [reply]
`TextInput::make(__('filament-panels::SettingsResource/forms.phone'))
                    ->tel()
                    ->regex('/^07\d{9}$/'),`

**Reply by pboivin** [reply, marked as answer]
It works for me, I'm getting the default laravel error message:
```
The test field format is invalid.
```
Did you customize the translations? Specifically `lang/xx/validation.php`

**Reply by Professor93M** [reply]
I published the validation translate file and the problem fix.

thank you.
